server/run-tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+LOG_LEVEL="test"
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+  --debug)
+    LOG_LEVEL="debug"
+    shift
+    ;;
+  esac
+  shift
+done
+
+export NODE_ENV="test"
+export SYNC_DATABASE="false"
+export SQL_DATABASE="dds_test"
+export LOG_LEVEL="$LOG_LEVEL"
+
+# Create test database if it doesn't exist.
+if [ "$(psql -tAc "SELECT 1 FROM pg_database WHERE datname='$SQL_DATABASE'" -h "$SQL_HOST" -p "$SQL_PORT" -U "$SQL_USER")" != '1' ]; then
```

Comment:
Ran into an interesting problem here. This command failed for me initially because installing postgres from homebrew didn't create a default database under my username. So when running initially I hit the error
`psql: error: FATAL:  database "jwileczek" does not exist`
I ultimately just needed to run `createdb` from the terminal to create a default database for my postgres, but I'm wondering if we should run a drop on the test database beforehand so we create it every time. It wouldn't matter that way if the user has the default user database setup before running tests. It would make us have to run migrations every time, but that may not be bad thing either?
`dropdb $SQL_DATABASE --if-exists`